waspc/data/Generator/templates/server/src/actions/_action.ts

```diff
@@ -7,7 +7,7 @@ import prisma from '../dbClient.js'
           consider in the future if it is worth removing this duplication. =}
 
 export default async function (args, context) {
-  return {= jsFn.importIdentifier =}(args as never, {
+  return ({= jsFn.importIdentifier =} as any)(args, {
```

Comment:
Ugly, but this assertion allows users not to specify `args` and `context` in their query/action implementations (if they don't need them)/

Reply:
We had to do that for server setup function, so it's a useful hack we have to use sometimes 👍